target/linux/bcm27xx/Makefile

```diff
@@ -8,8 +8,8 @@ include $(TOPDIR)/rules.mk
 ARCH:=arm
 BOARD:=bcm27xx
 BOARDNAME:=Broadcom BCM27xx
-FEATURES:=audio boot-part display ext4 fpu gpio rootfs-part rtc squashfs usb usbgadget
-SUBTARGETS:=bcm2708 bcm2709 bcm2710 bcm2711
+FEATURES:=audio boot-part display ext4 fpu gpio pci pcie rootfs-part rtc squashfs usb usbgadget
```

Comment:
PCIe is only present on Raspberry Pi 5, so `pci pcie` should go as `FEATURES+=pci pcie` in bcm2712/target.mk.
Otherwise you're also adding these features for RPi 1/2/3/4.

Reply:
done.